### DISCUSSION TITLE
Can I have the tiled windows be fullscreen?

### DISCUSSION THREAD

**Discussion by Ducky0Master** [i3/i3#5785] [Q&A] [answered]
When in fullscreen mode the entire screen is used(obviously). However, when in tilled mode, there is space in between the windows and the outer windows and the edge of the screen. How can I get rid of this space? I want an alone tilled window to look the same as a fullscreen window.

Edit: Actually, not entirely the same. I would like the bottom bar to be shown in tilled mode. I just don't want to see my desktop background on the edge of my screen and in between my tiles.

**Comment by orestisfl** [comment, marked as answer]
The space you are referring to is called `gaps` and is definitely not enabled by default. Try to remove any such configuration from your config file.
